submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/entities/GenericUser.kt

```diff
@@ -4,7 +4,7 @@ package ac.uk.ebi.biostd.itest.entities
  * Represent a Generic bio studies user.
  */
 object GenericUser {
-    const val username = "jhon_doe"
+    const val username = "jhon doe"
```

Comment:
Do we allow spaces in the user name? :thinking:

Reply:
username is actually user name, we need to change this convention in backend and frontend which was inherited from the previous system